### PR TITLE
Qt 6 update

### PR DESCRIPTION
**Pull request by TheComputerGuy96**

### CODE REVIEW

**Comment by AsciiWolf** [comment]
Thanks for this PR! The [net._86box._86Box.ROMs](https://github.com/flathub/net._86box._86Box.ROMs) Flatpak should also be updated.

File: net._86box._86Box.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 app-id: net._86box._86Box
 runtime: org.kde.Platform
-runtime-version: 5.15-23.08
+runtime-version: 6.6
```
Comment (AsciiWolf) [inline net._86box._86Box.yaml:3]: By the way, why using the 6.6 runtime instead of 6.7?

Comment (TheComputerGuy96) [reply net._86box._86Box.yaml]: The runtime is 6.8 now

**Comment by AsciiWolf** [comment]
I think that 6.7 would be better for now (and it should also use 24.08).

**Comment by TheComputerGuy96** [comment]
> I think that 6.7 would be better for now (and it should also use 24.08).

@AsciiWolf 6.7 still uses 23.08: https://invent.kde.org/packaging/flatpak-kde-runtime/-/blob/qt6.7/org.kde.Sdk.json.in#L8

**Comment by AsciiWolf** [comment]
Oops, I did not know that, thanks!

Anyway, I may be wrong, but I think that the 6.8 runtime is not yet considered stable.

**Comment by TheComputerGuy96** [comment]
> Anyway, I may be wrong, but I think that the 6.8 runtime is not yet considered stable.

@AsciiWolf The 6.6 runtime just got marked as EOL (so 6.7 will likely get the treatment when 6.9 gets released)

Also Qt 6.8 is an LTS release so it will get updates for longer

**Comment by TheComputerGuy96** [comment]
https://github.com/flathub/im.nheko.Nheko/pull/3#issuecomment-2454470675

**Comment by AsciiWolf** [comment]
Thanks!

**Pull request merged**